components/installer/deploy/installer/kyma.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM eu.gcr.io/kyma-project/installer:0.3.383
+
+LABEL source="git@github.com:kyma-project/kyma.git"
+
+COPY /kyma /kyma/injected
```

Comment:
All sources are copied?

Reply:
no, in our local scrips only resources and installation folders are copied. (in this POC i used local deployment for installer and kyma/resources and kyma/installation is already copied to installer folder)